src/System.Private.Xml/src/System.Private.Xml.builds

```diff
@@ -3,6 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Private.Xml.csproj" />
+    <Project Include="System.Private.Xml.csproj">
+      <TargetGroup>netcore50</TargetGroup>
+    </Project>
+    <Project Include="System.Private.Xml.csproj">
+      <TargetGroup>netcore50aot</TargetGroup>
+    </Project>
```

Comment:
Are you sure you need to do this? Looks like the default build is targeting netstandard1.3 which is compatible with netcore50 and netcore50aot, and it doesn't look like you are doing something special build-wise for the different TargetGroups, so I'm not sure I understand why we need to x-compile